utils/voter-bags/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "polkadot-voter-bags"
+version = "0.9.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+generate-bags = { git = "https://github.com/paritytech/substrate", branch = "zeke-voter-bags-module" }
```

Comment:
NOTE: this and all the bags-list stuff needs to get changed to the master branch